jsk_pr2_robot/jsk_pr2_startup/jsk_pr2_sensors/kinect_head.launch

```diff
@@ -85,6 +85,15 @@
         <remap from="~output" to="throttled/image_rect_color" />
         <param name="update_rate" value="$(arg throttle_rate)" />
       </node>
+      <node name="throttle_rgb_compressed"
+            pkg="nodelet" type="nodelet"
+            args="load jsk_topic_tools/LightweightThrottle /$(arg camera)/$(arg manager)"
```

Comment:
I suppose using `jsk_topic_tools/SynchronizedThrottle` to publish synchronized RGBG images.

Reply:
also camerainfos

Reply:
I don't know which is better.
Fetch and PR2 uses `LightweightThrottle` now, so we should try `LightweightThrottle` first,
and we can switch to `SynchronizedThrottle` when it is needed.

Reply:
I think that these patches use-cases are as follows.
- Visualize RGB-D images at remote PC
- Visualize PointCloud at remote PC. (Should be (a)synchronized)
- Run recognition program with the throttled image to reduce the resource of PC. (Should be (a)synchronized)

Considering the above, I think `SynchronizedThrottle` is better.
https://github.com/jsk-ros-pkg/jsk_hsr/blob/master/jsk_hsr_startup/launch/hsr_bringup.launch#L458-L472

> we can switch to SynchronizedThrottle when it is needed.

However, if you don't think you should do this right away, this change may be a good start.

Reply:
if it is so, let me open another PR to use `SychronizedThrottle`.
this PR purpose is to add missing throttle, not improving others.

Reply:
What is this PR's purpose? Please add the comment.

Reply:
Purpose of throttled image.

Reply:
for beginners

Reply:
throttle image is for rosbag.

Reply:
Isn't it better to write down the reason why you should use throttle images at the top comment of this PR?

Reply:
I added.